sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -146,6 +150,11 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override val annotations: HashMap<View, ViewAnnotationOptions>
+    get() = HashMap<View, ViewAnnotationOptions>().apply {
+      putAll(annotationsMap)
+    }
```

Comment:
maybe add private setter despite that modification will not update actual annotation options?